.golangci.yml

```diff
@@ -6,7 +6,6 @@ run:
 linters:
   disable-all: true
   enable:
-    - deadcode
```

Comment:
Deprecated in golangci-lint 1.49. `unused` is advised as the alternative and is already present in the `golangci.yaml`